ttnn/cpp/ttnn/operations/pool/downsample/downsample.hpp

```diff
@@ -4,31 +4,26 @@
 
 #pragma once
 
-#include "ttnn/tensor/types.hpp"
+#include "device/downsample_op.hpp"
 #include "ttnn/operations/core/core.hpp"
-
 #include "ttnn/run_operation.hpp"
 
-#include "device/downsample_op.hpp"
-
-
 namespace ttnn {
 namespace operations {
-namespace data_movement {
+namespace downsample {
```

Comment:
this should be pool

Reply:
Abhinav's PR is going to take care of this.